packages/sdk/src/ZSshClient.ts

```diff
@@ -40,6 +40,7 @@ export class ZSshClient extends AbstractRpcClient implements Disposable {
     private mPartialStderr = "";
     private mPartialStdout = "";
     private readonly mPromiseMap: Map<number, RpcPromise> = new Map();
+    // biome-ignore lint/correctness/noUnusedPrivateClassMembers: Linter Error: this.mRequestId is used
```

Comment:
Looks good for now, hopefully we can remove this comment once https://github.com/biomejs/biome/issues/6994 is fixed 🙏